packages/manager/modules/phone-prefix/README.md

```diff
@@ -0,0 +1,55 @@
+# manager-phone-prefix
```

Comment:
Do we need a specific module for this, can't it be added to `manager-components` ?

Reply:
It was discuss with others people.
We decide to go like this.

Reply:
Out of curiosity, what was the reason ? Seems a bit too much having a new module for this purpose